### PR TITLE
fix: Missing passing of `skip_tasks` flag

### PR DESCRIPTION
**Pull request by arnaubt**
Couldn't get the `--skip-flags` option working until I passed the option to the worker.

### CODE REVIEW

**Review by sisp — CHANGES_REQUESTED** [review]
Thanks for discovering and fixing this bug, @arnaubt! :pray: LGTM! :tada:

Just one request. It would be great to have a test, e.g. similar to:

https://github.com/copier-org/copier/blob/3f89b71cc6116e2f6dcda3ceb3ce4a41301895b8/tests/test_unsafe.py#L94-L113

Would you mind adding one?

**Comment by arnaubt** [comment]
@sisp Sure enough, test added! Let me know if it's good enough.

File: tests/test_unsafe.py [hunk 1]
```diff
@@ -113,6 +113,28 @@ def test_copy_cli(
...
+        ["copier", "copy", "--UNSAFE", *(["--skip-tasks"] if skip_tasks else []), str(src), str(dst)],
+        exit=False,
+    )
+
+    assert retcode == 0
+
+    if skip_tasks:
+        assert not (dst / "task.txt").exists()
+    else:
+        assert (dst / "task.txt").exists()
```
Comment (sisp) [inline tests/test_unsafe.py:135]: I think we can simplify this a bit like this:

```suggestion
    assert (dst / "task.txt").exists() is (not skip_tasks)
```

WDYT?

File: tests/test_unsafe.py [hunk 2]
```diff
@@ -113,6 +113,28 @@ def test_copy_cli(
...
+    build_file_tree(
+        {(src / "copier.yaml"): yaml.safe_dump({"_tasks": ["touch task.txt"]})}
+    )
+    _, retcode = CopierApp.run(
+        ["copier", "copy", "--UNSAFE", *(["--skip-tasks"] if skip_tasks else []), str(src), str(dst)],
+        exit=False,
+    )
+
+    assert retcode == 0
+
```
Comment (sisp) [inline tests/test_unsafe.py:131]: Just a minor cosmetic thing: No need for empty lines here.

```suggestion
    assert retcode == 0
```

**Review by sisp — CHANGES_REQUESTED** [review]
LGTM, just some minor requests. Sorry that I was a bit imprecise, I was thinking to add the test in `tests/test_tasks.py` perhaps below

https://github.com/copier-org/copier/blob/3f89b71cc6116e2f6dcda3ceb3ce4a41301895b8/tests/test_tasks.py#L69-L82

Could you please move it there?

**Comment by arnaubt** [comment]
Test moved and suggestions applied, thanks!

File: tests/test_tasks.py [hunk 3]
```diff
@@ -82,6 +84,24 @@ def test_copy_skip_tasks(template_path: str, tmp_path: Path) -> None:
...
+) -> None:
+    src, dst = map(tmp_path_factory.mktemp, ["src", "dst"])
+    build_file_tree(
+        {(src / "copier.yaml"): yaml.safe_dump({"_tasks": ["touch task.txt"]})}
+    )
+    _, retcode = CopierApp.run(
+        ["copier", "copy", "--UNSAFE", *(["--skip-tasks"] if skip_tasks else []), str(src), str(dst)],
+        exit=False,
+    )
+
```
Comment (sisp) [inline tests/test_tasks.py:100]: I forgot to remove the other empty line. Let me just remove it quickly myself.

```suggestion
```

**Review by sisp — APPROVED** [review]
LGTM! :tada: Thanks again, @arnaubt! :pray:

**Comment by sisp** [comment]
@yajo is creating releases for us. Would you be open to and available for making a quick patch release, @yajo?

**Comment by yajo** [comment]
Released!
https://github.com/copier-org/copier/releases/tag/v9.3.1

**Comment by sisp** [comment]
Thank you very much! :pray:

**Pull request merged**